### ISSUE TITLE
<html> element must have a lang attribute

### ISSUE THREAD

**Issue by maxdevjs** [processing/p5.js-web-editor#1192]
#### Nature of issue?

- Existing feature enhancement

#### Feature enhancement details:

To address and fix accessibility html lang attribute violation: https://github.com/maxdevjs/p5.js-web-editor/commit/4220de6a780e9d9fad89d4963f467a7e5d2fe3f8

I first of all create an issue

- because I do not know if there is a good reason this was not addressed already
- I assume as primary language of the document English 

Also, same violation is valid for the preview IFrame.

**Comment by catarak** [comment]
thanks for submitting! yes, this is a bug, and yes, this should be fixed.

**Comment by maxdevjs** [comment]
In https://github.com/maxdevjs/p5.js-web-editor/commit/4220de6a780e9d9fad89d4963f467a7e5d2fe3f8 I address the violation for main `html` tag.

Can I open a PR? If so, it should be done from my master (after merging the fix branch), right?

**Comment by maxdevjs** [comment]
Also... The same violation is generated by the review iFrame, but at this point I am not sure how the iFrame `html` element could be fixed.

**Comment by catarak** [comment]
@maxdevjs yes, that is the right file to fix! i think you'll also need to update the 404 page (https://github.com/processing/p5.js-web-editor/blob/master/server/views/404Page.js). to change the iframe, you'll need to change:
https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/reducers/files.js#L13

lastly, there are a few scripts that automatically fetch sketches from github and upload them to the web editor, which could be updated, specifically https://github.com/processing/p5.js-web-editor/blob/feature/public-api/server/scripts/examples-gg-latest.js#L23 and https://github.com/processing/p5.js-web-editor/blob/feature/public-api/server/scripts/examples.js#L12. 

feel free to submit separate pull requests if this is too much stuff to change at once, or you don't want to change stuff that doesn't make sense!
